docs/README.md

```diff
@@ -0,0 +1,18 @@
+@fluencelabs/fluence / [Exports](modules.md)
+
+# Fluence JS
+
+To start developing applications with Fluence JS refer to the official [gitbook page](https://doc.fluence.dev/docs/js-sdk)
+
+Fluence JS is the implementation of a Fluence protocol for JS-based environments. It can be used to connect browsers, Node.js applications and so on to the Fluence p2p network.
+
+Similar to reference node implementation it provides:
+
+- Peer-to-peer communication layer
+- Marine interpreter
+- Aqua VM
+- Builtin services
```

Comment:
Marine is a WebAssembly runtime, AFAIK Fluence JS doesn't use it.

Also I'd add a link to the list of Builtin Services that exist in Fluence JS
```suggestion
- Peer-to-peer communication layer (via [js-libp2p](https://github.com/libp2p/js-libp2p))
- [Aqua VM](https://github.com/fluencelabs/aquavm)
- Builtin services
```